.github/workflows/ci-app-tests.yml

```diff
@@ -10,6 +10,7 @@ on:
       - ".github/workflows/ci-app-tests.yml"
       - "src/lightning_app/**"
       - "tests/tests_app/**"
+      - "tests/tests_app_examples/**"
```

Comment:
@Borda Why did you add this if this workflow does not run the examples?

Reply:
I'm not @Borda, but the reason is simple - `checkgroup` expected these tests, but they weren't run. It's more like a bug within `checkgroup`

Reply:
Okay, so we should fix the config instead, not add this here

Reply:
Done in #15332

Reply:
> `checkgroup` expected these tests, but they weren't run. It's more like a bug within `checkgroup`

yep :innocent: